File: sdks/php/src/Wallets/Types/GetTransactionsWalletsRequestTransactionStatus.php
<?php

namespace NovaBilling\Wallets\Types;

enum GetTransactionsWalletsRequestTransactionStatus: string
{
    case Purchased = "PURCHASED";
    case Granted = "GRANTED";
    case Voided = "VOIDED";
    case Invoiced = "INVOICED";
}

File: sdks/ruby/lib/novabilling/wallets/types/get_transactions_wallets_request_transaction_status.rb
# frozen_string_literal: true

module Novabilling
  module Wallets
    module Types
      module GetTransactionsWalletsRequestTransactionStatus
        extend Novabilling::Internal::Types::Enum

        PURCHASED = "PURCHASED"
        GRANTED = "GRANTED"
        VOIDED = "VOIDED"
        INVOICED = "INVOICED"
      end
    end
  end
end
